apps/token/src/components/vega-wallet/hooks.ts

```diff
@@ -90,7 +90,9 @@ export const usePollForDelegations = () => {
             ).map((e) => e.node);
             setAccounts(
               accounts
-                .filter((a) => a.type === AccountType.ACCOUNT_TYPE_GENERAL)
+                .filter(
```

Comment:
nit (non-blocking): i know it's not a change in the implementation, usually i think it's nicer to do a reduce over a map + filter, to reduce the number of iterations. not sure how big these arrays get, if >100, probably doesn't matter too much.

Reply:
Also noted. It *shouldn't* get that big, but over time it is possible that a party could have that many accounts.